### ISSUE TITLE
DataTable/Paginator pt: paginatorContainer not work.

### ISSUE THREAD

**Issue by JoseAntonioNavasSolutia** [primefaces/primevue#6531]
### Describe the bug

I need to add an aria-label to the nav element of the pagination but it does not work.

### Reproducer

https://stackblitz.com/edit/7evhky?file=src%2FApp.vue

### PrimeVue version

3.52.0

### Vue version

3.x

### Language

ES6

### Build / Runtime

Nuxt

**Comment by tugcekucukoglu** [comment]
Correct name `pcPaginator`.
